src/app/containers/ConsentBanner/CanonicalLogic/index.js

```diff
@@ -14,8 +14,29 @@ const PRIVACY_COOKIE_PREVIOUS_VALUES = ['0', '1'];
 
 const onClient = typeof window !== 'undefined';
 
+const removeTLDRestriction = domain => {
```

Comment:
Can this be renamed to `removeTopLevelDomainRestriction`? (just to avoid more abbreviations :) )